dashboard/app/static/coverage.js

```diff
@@ -1,8 +1,8 @@
 // Copyright 2024 syzkaller project authors. All rights reserved.
 // Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
 
-$(document).ready(initTogglers());
-$(document).ready(initUpdateForm);
+initTogglers();
```

Comment:
Why drop `$(document).ready`?

Reply:
initTogglers works correctly w/o it. You can see the bug there.
initUpdateForm w/o `$(document).ready` doesn't require the document but anyway is waiting for it. You can check the UI. It may be the side effect of how I inject jquery there.

I don't see any value in keeping it there.

Reply:
But shouldn't we better still wait until DOM is ready before we start manipulating it?